include/profiler.h

```diff
@@ -198,7 +234,7 @@ static inline void profiler_log_add_mem_address(struct log_event_buf *buf,
 /** @brief Send data from the buffer to the host.
  *
  * This function only sends data that is already stored in the buffer.
- * Use @ref profiler_log_encode_u32 or @ref profiler_log_add_mem_address
+ * Use @ref profiler_log_encode_... or @ref profiler_log_add_mem_address
```

Comment:
This  `@ref` would not work. You can either remove the reference or list all of the functions with separate refs here